core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -168,6 +168,270 @@
   <script src="/extensions/objects/templates/SetOfUnicodeStringEditorDirective.js"></script>
   <script src="/extensions/objects/templates/UnicodeStringEditorDirective.js"></script>
 
+  <script src="/templates/dev/head/components/attribution_guide/AttributionGuideDirective.js"></script>
+  <script src="/templates/dev/head/components/embed_modal/ExplorationEmbedButtonService.js"></script>
+  <script src="/templates/dev/head/components/forms/HtmlSelectDirective.js"></script>
```

Comment:
is this an expected change?

Reply:
Hey Sandeep sorry - this isn't expected. I was trying to intermittently push a commit because I am not able to run my stack or any frontend tests after the webpack PR was checked in. I am trying to fix those and then this PR should reflect that!